adlfs/spec.py

```diff
@@ -1769,20 +1772,23 @@ def connect_client(self):
                 f"Unable to fetch container_client with provided params for {e}!!"
             )
 
-    async def _async_fetch_range(self, start: int, end: int, **kwargs):
+    async def _async_fetch_range(self, start: int, length: int = None, **kwargs):
```

Comment:
This function is ultimately overriding [`_fetch_range` in `fsspec.spec.AbstractBufferedFile`](https://github.com/intake/filesystem_spec/blob/a5fa9babfe4583cf21932bd18bdae8a6e13cc3a7/fsspec/spec.py#L1445-L1447) right? If so, I guess the function signature needs to be kept `start` and `end` here, since that is what `fsspec` assumes?